src/types/data-store.ts

```diff
@@ -15,39 +15,29 @@ export interface DataStore {
   close(): Promise<void>;
 
   /**
-   * Puts the given data in store.
-   * It is expected that the CID of the dataStream matches the given dataCid.
-   * The returned dataCid and returned dataSize will be verified against the given dataCid (and inferred dataSize).
-   * @param messageCid CID of the message that references the data.
-   * @returns The CID and size in number of bytes of the data stored.
+   * Stores the given data.
+   * @param recordId The logical ID of the record that references the data.
+   * @param dataCid The IPFS CID of the data.
```

Comment:
I really meant it as IPFS CID as that's the hash we create using the IPFS importer. While an IPFS CID is an IPLD CID, I  used IPFS CID because:

1. It is arguably more generally known.
1. It is more specific.

Reply:
Makes sense as it's more specific.